src/Tizen.NUI/src/public/BaseComponents/View.cs

```diff
@@ -2307,21 +2306,6 @@ public Vector2 UpdateSizeHint
             }
         }
 
-        /// This will be public opened in tizen_next after ACR done. Before ACR, need to be hidden as inhouse API.
```

Comment:
Can you confirm this property was not used? Hidden api(EditorBrowsableState.Never) are also can be used. Just removing might be dangerous.